content/rs/installing-upgrading/_index.md

```diff
@@ -342,37 +312,6 @@ To install with an answer file:
     ```sh
     ./install.sh -c /home/user/answers
     ```
-    {{% /expand %}}
-
-    When install suceeds, the IP address of the admin console is shown:
-
-    ```sh
-    Summary:
-    -------
-    ALL TESTS PASSED.
-    2017-04-24 10:54:15 [!] Please logout and login again to make
-    sure all environment changes are applied.
-    2017-04-24 10:54:15 [!] Point your browser at the following
-    URL to continue:
-    2017-04-24 10:54:15 [!] https://<your_ip_here>:8443
-    ```
-
-    Redis Software is now installed on the node.
-    Repeat this process for each node in the cluster.
-
-1. [Create]({{< relref "/rs/administering/new-cluster-setup.md" >}})
-    or [join]({{< relref "/rs/administering/adding-node.md" >}}) an existing RS cluster.
-1. [Create a database]({{< relref "/rs/administering/creating-databases/_index.md" >}}).
-
-    For geo-distributed Active-Active replication, create an [Active-Active]({{< relref "/rs/administering/creating-databases/create-active-active.md" >}}) database.
-
-## Set up the new cluster
-
-{{< embed-md "new-cluster-embed.md" >}}
-
-## Add nodes to the cluster
-
-{{< embed-md "adding-node-embed.md" >}}
 
 ## Next steps
```

Comment:
Link to "Add users" does not go to the right place (redirects me to the same page /rs/installing-upgrading/ but maybe it's only because it's staging?)
I also think we might want to link to 2 places:
rs/security/passwords-users-roles/
rs/security/admin-console-security/user-security/

Reply:
Updated.